dev/scripts/build.ts

```diff
@@ -3,46 +3,139 @@ import { join } from "node:path"
 import { readJson, writeJson } from "../attest/src/fs.js"
 import { shell } from "../attest/src/shell.js"
 import { repoDirs } from "./common.js"
+import { rewritePaths } from "./overwrite.js"
+import type { ReplacementDictionary } from "./overwrite.js"
 
 const packageRoot = process.cwd()
 const outRoot = join(packageRoot, "dist")
 const packageJson = readJson(join(packageRoot, "package.json"))
+const tempTsConfigBaseName = "tsconfig.temp"
+const tempTsConfigPath = join(packageRoot, `${tempTsConfigBaseName}.json`)
 
-const buildFormat = (module: "commonjs" | "esnext") => {
-    const outDir = join(outRoot, module === "commonjs" ? "cjs" : "mjs")
-    writeJson(tempTsConfig, {
-        ...tsConfigData,
-        include: ["src"],
+const nuke = (target: string) =>
```

Comment:
Completely my fault for not having mentioned this earlier in the process, but you should check out `attest/src/fs` in general and particularly `getSourceFileEntries` should make this process much easier.